src/sensors/mpu9250sensor.cpp

```diff
@@ -116,12 +116,25 @@ void MPU9250Sensor::motionSetup() {
         m_Logger.error("DMP Initialization failed (code %d)", devStatus);
     }
 #else
+    // NOTE: could probably combine these into less total writes, but this should work, and isn't time critical.
+    imu.setAccelFIFOEnabled(true);
+    imu.setXGyroFIFOEnabled(true);
+    imu.setYGyroFIFOEnabled(true);
+    imu.setZGyroFIFOEnabled(true);
+    imu.setSlave0FIFOEnabled(true);
+
+    // TODO: set a rate we prefer instead of getting the current rate from the device.
+    deltat = 1.0 / 1000.0 * (1 + imu.getRate());
```

Comment:
I could be wrong, but we don't appear to have ever been setting a specific rate anywhere? so I'm querying from the IMU what rate is set and computing timings based off of that, but we can change this to set it to a rate of our choosing.